clusterloader2/pkg/measurement/common/wait_for_rc_pods.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+const (
+	defaultWaitForRCPodsTimeout  = 60 * time.Second
+	defaultWaitForRCPodsInterval = 5 * time.Second
+)
+
+func init() {
+	measurement.Register("WaitForRunningRCs", createWaitForRunningRCsMeasurement)
+}
+
+func createWaitForRunningRCsMeasurement() measurement.Measurement {
```

Comment:
Can you make this file generic to work with different kinds of controllers (e.g RC, RS, deployment, job, etc)? Because we don't want to have one copy of this implementation for each of them.

Here's a reference to how we're doing it in a generic way - https://github.com/kubernetes/kubernetes/blob/7b2fa0a21f2f82dcc9b424d8f0f634e0263cb403/test/e2e/framework/util.go#L2892

Reply:
Also after making that change, I suggest renaming this file to `wait_for_controlled_pods.go`.

Reply:
Done.